cmd/stunstamp/stunstamp.go

```diff
@@ -534,7 +578,12 @@ func remoteWriteTimeSeries(client *remoteWriteClient, tsCh chan []prompb.TimeSer
 }
 
 func main() {
+	flag.Var(&flagDstPorts, "dst-port", "destination ports to monitor; may be specified multiple times")
 	flag.Parse()
+	if len(flagDstPorts) < 1 {
```

Comment:
"== 0" is fine, len will never be negative 🙂

Reply:
updated in https://github.com/tailscale/tailscale/pull/12356/commits/58863f931f081c5a5d6ff886660bd66d76a6ecc0